packages/teleport-generator-core/src/validator/index.ts

```diff
@@ -62,14 +62,14 @@ export default class Validator {
   }
 
   public validateProjectContent(input: ProjectUIDL): ValidationResult {
-    const verifyRoutes = checkRouteDefinition(input)
+    const verifyRoutes = utils.checkRouteDefinition(input)
     let errors = verifyRoutes.errors
 
     if (verifyRoutes.errors.length === 0) {
-      const verifyRootComponent = checkRootComponent(input)
+      const verifyRootComponent = utils.checkRootComponent(input)
 
-      const verifyComponentNaming = checkComponentNaming(input)
-      const verifyComponentExistence = checkComponentExistence(input)
+      const verifyComponentNaming = utils.checkComponentNaming(input)
+      const verifyComponentExistence = utils.checkComponentExistence(input)
```

Comment:
once all the functions returns an array of errors, you can spread them together to form a single array of error messages